src/styleguide/_patterns/01-atoms/06-buttons/08-toggle-menu.mustache

```diff
@@ -1,13 +1,4 @@
-<p class="is-light">
-  <button type="button" class="grav-c-toggle-menu" aria-pressed="{{control.pressed}}" aria-label="{{control.label}}">
-   <span class="grav-c-toggle-menu__icon"></span>
-  </button>
-</p>
-
-<h4>Dark background</h4>
 
-<p class="sg-dark-background">
-  <button type="button" class="grav-c-toggle-menu" aria-pressed="{{control.pressed}}" aria-label="{{control.label}}">
-    <span class="grav-c-toggle-menu__icon"></span>
-  </button>
-</p>
+<button type="button" class="grav-c-toggle-menu" aria-pressed="{{control.pressed}}" aria-label="{{control.label}}">
```

Comment:
Now that this template has been trimmed back to just the actual component, could you please update the `molecules-nav-menu` to include this pattern instead of containing a copy-paste of its code?

Thanks!

Reply:
done & done.